src/shared/SellerProfile/ReviewHeader.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import classNames from 'classnames';
 import get from 'lodash/get';
 
+import profile from './profile.css'; // eslint-disable-line no-unused-vars
```

Comment:
just thinking in the future you might want to call this something like `sheet` instead of `profile` given that your style includes are not declarative